test/scripts/run_test.py

```diff
@@ -47,47 +49,58 @@ def analyse_instrumentation_test_result(result):
     return ok_num
 
 
-def analyse_unit_test_result(result):
-    total_num = 0
-    fail_num = 0
+def analyse_unit_test_result(result, mode, module):
```

Comment:
it's better take a `result_file.xml` as this function's parameter

Reply:
done